src/models/test/models_test.ts

```diff
@@ -280,6 +280,36 @@ describe('GenerativeModel startChat', () => {
 
     expect(chat).toBeInstanceOf(ChatSession);
   });
+  it('returns ChatSession when pass an arg with generationConfig including responseMimeType', async () => {
+    const expectedResult = TEST_MODEL_RESPONSE;
+    const fetchResult = Promise.resolve(
+      new Response(JSON.stringify(expectedResult), fetchResponseObj)
+    );
+    const fetchSpy = spyOn(global, 'fetch').and.returnValue(fetchResult);
+
+    const model = new GenerativeModel({
+      model: 'gemini-pro',
+      project: PROJECT,
+      location: LOCATION,
+      googleAuth: FAKE_GOOGLE_AUTH,
+    });
+    const chat = model.startChat({
+      generationConfig: {
+        ...TEST_GENERATION_CONFIG,
+        responseMimeType: 'application/json',
+      },
+    });
+
+    expect(chat).toBeInstanceOf(ChatSession);
+
+    const req = 'How are you doing today?';
+    await chat.sendMessage(req);
+    const expectedBody =
+      '{"contents":[{"role":"user","parts":[{"text":"How are you doing today?"}]}],"generationConfig":{"candidateCount":1,"stopSequences":["hello"],"responseMimeType":"application/json"}}';
+    // @ts-ignore
+    const actualBody = fetchSpy.calls.allArgs()[0][1].body;
+    expect(actualBody).toEqual(expectedBody);
```

Comment:
can you create a separate test case for this assertion? L303 and L311 are two different test intents. Same comment applies for L515.

Reply:
@yyyu-google I pushed the commit to separate the test cases at [bb069aa](https://github.com/googleapis/nodejs-vertexai/pull/364/commits/bb069aa7160f800b2b7235a80f934f750c337d38).